prql-compiler/src/semantic/context.rs

```diff
@@ -602,14 +602,6 @@ impl std::fmt::Display for DeclKind {
     }
 }
 
-impl std::fmt::Debug for TableDecl {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        let json = serde_json::to_string(self).unwrap();
-        let json = serde_json::from_str::<serde_json::Value>(&json).unwrap();
-        f.write_str(&serde_yaml::to_string(&json).unwrap())
-    }
-}
```

Comment:
I'm using this Debug impl for more concise printing, as default Debug is quite large. But I can re-add when needed.